pages/linux/shar.md

```diff
@@ -0,0 +1,28 @@
+# shar
```

Comment:
If the same command is available in more than 1 platform should be placed in the `common/` directory. Please move.

Reply:
I've resolved the issue